### PR TITLE
SWATCH-1997: Prometheus Wiremock tests need to run on a randomized port

### PR DESCRIPTION
**Pull request by Sgitario**
Jira issue: [SWATCH-1997](https://issues.redhat.com/browse/SWATCH-1997)

## Description
Running the tests with org.gradle.parallel=true (which is the default for our project) results in test failures as the tests contend over port 8101.

## Testing
Build the project using: `./gradlew clean build` (it's using parallel by default). All the tests are passing now.

### CODE REVIEW

File: swatch-contracts/build.gradle [hunk 1]
```diff
@@ -93,9 +93,3 @@ openApiGenerate {
 }
 
 sourceSets.main.java.srcDirs += ["${buildDir}/generated/src/gen/java"]
-
-tasks.register("wiremock", JavaExec) {
```
Comment (Sgitario) [inline swatch-contracts/build.gradle:97]: This task was not used.

File: swatch-contracts/src/test/java/com/redhat/swatch/contract/resource/WireMockResource.java [hunk 2]
```diff
@@ -48,18 +47,6 @@ public class WireMockResource
   public static final String STORE_PASSWORD = "password";
   private WireMockServer wireMockServer;
 
-  public static void main(String[] args) {
```
Comment (Sgitario) [inline swatch-contracts/src/test/java/com/redhat/swatch/contract/resource/WireMockResource.java:51]: This was not used.

File: swatch-contracts/src/test/java/com/redhat/swatch/contract/resource/WireMockResource.java [hunk 3]
```diff
@@ -86,10 +73,10 @@ public Map<String, String> start() {
   private void stubApis() {
     stubForRhPartnerApi(wireMockServer);
     stubForInternalSubscriptionService(wireMockServer);
-    stubForSubscriptioinService(wireMockServer);
+    stubForSubscriptionService(wireMockServer);
```
Comment (Sgitario) [inline swatch-contracts/src/test/java/com/redhat/swatch/contract/resource/WireMockResource.java:76]: Fixed typo.

File: swatch-producer-aws/build.gradle [hunk 4]
```diff
@@ -67,9 +66,3 @@ openApiGenerate {
 tasks.openApiGenerate.inputs.file("../swatch-core/schemas/billable_usage.yaml")
 
 sourceSets.main.java.srcDirs += ["${buildDir}/generated/src/gen/java"]
-
-tasks.register("wiremock", JavaExec) {
```
Comment (Sgitario) [inline swatch-producer-aws/build.gradle:71]: this task was not used.

**Review by kahowell — APPROVED** [review]

**Pull request merged**